modules/videoio/src/cap_ffmpeg_impl.hpp

```diff
@@ -927,7 +927,6 @@ class InternalFFMpegRegister
         if(!threadSafe)
             lock.lock();
         static InternalFFMpegRegister instance;
-        initLogger_();  // update logger setup unconditionally (GStreamer's libav plugin may override these settings)
```

Comment:
I take a look on the comment.

It comes from this commit: https://github.com/opencv/opencv/pull/19661/commits/7baf2e8789dfe753bbcb5707eedefdb02433ea76

which commit message says "repair FFmpeg logger settings on each .open() call"

@mshabunin Do you remember which reproducer we had for this case?

Reply:
> @mshabunin Do you remember which reproducer we had for this case?

No, I don't :slightly_frowning_face:
PR was #19661